config/initializers/cors.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.config.middleware.insert_before 0, Rack::Cors do
   allow do
-    origins 'http://localhost:3000' # Add any additional origins as needed
+    origins 'http://localhost:3001' # Add any additional origins as needed
```

Comment:
- [ ] These are instructions for the frontend, not for the backend:

![image](https://github.com/henokkhm/workspace-share-backend-microverse-capstone/assets/57421823/4fe18c84-8775-401c-8603-a32ef83dac47)

Kindly: 

- Remove the frontend instructions from the backend readme, to prevent confusion.
- Include specific instructions about how to setup and run the backend server (bundle install, rails db:create, rails db:migrate, rails db:seed, etc)

That way, all your users will be able to run your project correctly. 👍